modules/@ergonode/authentication/src/locales/en_GB.json

```diff
@@ -41,7 +41,7 @@
             "checkEmail": {
                 "title": "Check your e-mail",
                 "info": "The link to reset your password will be sent to the provided e-mail address.",
-                "infoText": "If it does not arrive, please ",
+                "infoText": "If it does not arrive, please [[infoLink]]",
```

Comment:
Is it some pattern to match links? What about passing link as a different param, extra regex would not be needed.